### PR TITLE
use ko to build svc container images

### PR DESCRIPTION
**Pull request by SoMuchForSubtlety**
Changes the CD action to use ko[^1], so we don't need qemu and can use the go build cache.

### TODO:
- [x] use a custom base image
- [x] test the action
- [x] remove the svc-ci action?
- [x] remove the old dockerfile?
- [x] link ghcr.io/memelabs/strims/ffmpeg to this repo
- [x] fix ffmpeg apko action or switch to qemu + buildx

[^1]: https://github.com/google/ko

### CODE REVIEW

**Comment by jbpratt** [comment]
@SoMuchForSubtlety you'll need to disable the PR event trigger, I don't think it's allowed to push (and shouldn't be)

**Comment by jbpratt** [comment]
~4m build @slugalisk :partying_face:

**Comment by SoMuchForSubtlety** [comment]
> ~4m build @slugalisk 🥳

~1m 😏

File: infra/hack/kubernetes/strims/30-svc.yaml [hunk 1]
```diff
@@ -79,9 +79,8 @@ spec:
       - name: strims
         image: ghcr.io/memelabs/strims/svc:latest
         imagePullPolicy: Always
-        command: ["sh", "-c"]
         args:
-        - /app/svc run -config /etc/strims/config.yaml -host-ip "$(cat /node-labels/public-ip)";
+        - run -config /etc/strims/config.yaml -host-ip "$(cat /node-labels/public-ip)";
```
Comment (jbpratt) [inline infra/hack/kubernetes/strims/30-svc.yaml:83]: I think this is required for the `"$(cat /node-labels/public-ip)";` hack

Comment (SoMuchForSubtlety) [reply infra/hack/kubernetes/strims/30-svc.yaml:83]: Ah, I'll need to figure out where ko puts the binary.

**Comment by slugalisk** [comment]
> > ~4m build @slugalisk 🥳
> 
> ~1m 😏

🤯

**Pull request merged**